.ats/main.yaml

```diff
@@ -0,0 +1,3 @@
+smoke-tests-cluster-type: kind
```

Comment:
For now we just executed the basic smoke test scenario. We can add the upgrade test when more versions are out. It won't work otherwise.